firefox_puppeteer/ui/toolbars.py

```diff
@@ -144,13 +165,21 @@ def history_drop_marker(self):
         """
         return self.urlbar.find_element(By.ANON_ATTRIBUTE, {'anonid': 'historydropmarker'})
 
-    @use_class_as_property('ui.toolbars.IdentityPopup')
+    @property
     def identity_popup(self):
         """Provides utility members for accessing and manipulating the
         locationbar.
 
         See the :class:`IdentityPopup` reference.
         """
+        if not self._identity_popup:
+
+
+        
+        	from .toolbars import IdentityPopup
+        	self._identity_popup = IdentityPopup(lambda: self.marionette, self.window)
```

Comment:
It looks like that you are using tabs for indentation. Please disable that feature of your text editor and instead use 4 blanks. Also remove the empty lines above.